rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -152,8 +156,54 @@ If the resolved transaction contents is not the latest version, this may indicat
 
 Since the transaction content is detached from the transaction itself and referred to by hash, the transaction content MUST be hashed and compared to the hash specified in the transaction, to assert that the retrieved transaction content is actually the expected transaction content.
 
+### 3.8 Private Transactions
+
+Private transactions are transactions that contain sensitive content, intended for a single specific, or multiple recipients.
+Private transactions MUST be added to the DAG like non-private transactions.
+The recipients MUST be specified as array that contains the node DIDs of the recipients, encrypted separately for each recipient.
+To construct the data to be encrypted, the plaintext recipients MUST be joined by a newline (`\n`)
+
+The recipient list MUST be encrypted for each recipient and the ciphertext specified as array as `pal` header in the JWS.
+
+Given 2 recipients `did:nuts:recipient-A` and `did:nuts:recipient-B`:
+
+```
+encoded_recipients = join("did:nuts:recipient-A", "did:nuts:recipient-B", "\\n")
```

Comment:
This could be an JSON array of strings (where each item contains base64 encoded cipher-text)

Reply:
The `\\n` feels a bit weird....

Reply:
This makes sense now after getting up to date with @reinkrul (I thought each address was encrypted but it's actually the entire list of participants that get encrypted per participant)

Reply:
We need a way to encode a list of DIDs to a byte array, so it can be encrypted. Choices are JSON (feels a bit weird and heavy weight, marshalling a Go `[]byte` to JSON, just to encrypt it) or a custom encoding. I figured the simplest and most space-efficient way is just separating them with a character that can't be used in DIDs, so I chose a newline.